[Details Revamp] Remove the Three Dot Menu on Affected Reports

cdOut opened the pull request:
<!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->

### Details
<!-- Explanation of the change or anything fishy that is going on -->

This is a final PR for the Details Revamp task which aims at removing the Three Dot Menu from reports affected by the revamp. It also fixes a bug regarding the Hold promoted action not displaying properly for single expenses.

### Fixed Issues
<!---
1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing. For example, `$ https://github.com/Expensify/App/issues/<issueID>`.
2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).  For example, `PROPOSAL: https://github.com/Expensify/App/issues/<issueID>#issuecomment-1369752925`

Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
It MUST be an entire link to the github issue and your comment proposal ; otherwise, the linking and its automation will not work as expected.

Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):

$ https://github.com/Expensify/App/issues/<issueID>
$ https://github.com/Expensify/App/issues/<issueID(comment)>

Do NOT only link the issue number like this: $ #<issueID>
--->
$ https://github.com/Expensify/App/issues/42080
PROPOSAL:


### Tests
<!---
Add a numbered list of manual tests you performed that validates your changes work on all platforms, and that there are no regressions present.
Add any additional test steps if test steps are unique to a particular platform.
Manual test steps should be written so that your reviewer can repeat and verify one or more expected outcomes in the development environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->

1. Create a workspace
2. Go into a room of said workspace and confirm that the three dot menu in the header is missing
3. Create two expenses for the workspace
4. Go into the report view containing both those expenses and confirm the three dot menu is missing
5. Go into one of the expenses and confirm that the three dot menu is missing
6. If you have an access to an account with a Concierge guide assigned, please check whether the button for `Book a call` works properly, since it has been removed from the three dot menu in that chat.

- [x] Verify that no errors appear in the JS console

### Offline tests
<!---
Add any relevant steps that validate your changes work as expected in a variety of network states e.g. "offline", "spotty connection", "slow internet", etc. Manual test steps should be written so that your reviewer and QA testers can repeat and verify one or more expected outcomes. If you are unsure how the behavior should work ask for advice in the `#expensify-open-source` Slack channel.
--->

### QA Steps
<!---
Add a numbered list of manual tests that can be performed by our QA engineers on the staging environment to validate that your changes work on all platforms, and that there are no regressions present.
Add any additional QA steps if test steps are unique to a particular platform.
Manual test steps should be written so that the QA engineer can repeat and verify one or more expected outcomes in the staging environment.

For example:
1. Click on the text input to bring it into focus
7. Upload an image via copy paste
8. Verify a modal appears displaying a preview of that image
--->

- [x] Verify that no errors appear in the JS console

### PR Author Checklist
<!--
This is a checklist for PR authors. Please make sure to complete all tasks and check them off once you do, or else your PR will not be merged!
-->

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>


https://github.com/Expensify/App/assets/88325488/8c33d26e-1d46-4d85-b7ba-8125da247947


</details>

<details>
<summary>iOS: Native</summary>


https://github.com/Expensify/App/assets/88325488/5860c59c-e9c7-4acd-af94-9fd7f154ebab


</details>

<details>
<summary>iOS: mWeb Safari</summary>


https://github.com/Expensify/App/assets/88325488/56a45763-f1a1-4f9a-889c-3e13052b8113


</details>

<details>
<summary>MacOS: Chrome / Safari</summary>


https://github.com/Expensify/App/assets/88325488/6e83fd81-152b-4b9c-9b3d-711c0133fb2c


</details>

<details>
<summary>MacOS: Desktop</summary>


https://github.com/Expensify/App/assets/88325488/413039ed-57d1-4b52-ba3c-8932f54622be


</details>

cdOut committed:
remove three dot menu from affected pages

cdOut committed:
clean up unused imports and vars

cdOut commented:
cc: @grgia

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal

cdOut committed:
fix prettier

cdOut committed:
move the Book a call action into report details

cdOut committed:
remove Book a call from HeaderView

cdOut committed:
fix HeaderView prettier

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal

cdOut committed:
remove code from MoneyReportHeader

cdOut committed:
fix lint errors

cdOut committed:
fix prettier

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal

trjExpensify commented:
@parasharrajat from [here](https://expensify.slack.com/archives/C02NK2DQWUX/p1719917745648169?thread_ts=1719915772.178069&cid=C02NK2DQWUX), please give this a review now. Thanks!

cdOut committed:
move Book a call from ReportDeailsPage into ProfilePage

cdOut committed:
fix prettier

cdOut commented:
> BUG: I don't see Book a call button on details page.

I've moved the code a little bit so the button for Concierge Book a call action should now properly display.

> Have we moved all the options related functionality to the details page?

Yes, I think so. Did you find a functionality that was removed and is not in the ReportDetailsPage?

> Why did we remove this menu? It shows information about held transactions when users open one.

This is a correct observation, I'll add it back.

cdOut committed:
remove import issues and fix lint

cdOut committed:
revert hold menu removal from MoneyRequestHeader

cdOut committed:
fix prettier issues

cdOut commented on src/components/MoneyRequestHeader.tsx:
```diff
@@ -144,163 +93,76 @@ function MoneyRequestHeader({report, parentReportAction, policy, shouldUseNarrow
-                shouldEnableNewFocusManagement
-            />
-            {isSmallScreenWidth && shouldShowHoldMenu && (
-                <ProcessMoneyRequestHoldMenu
```
Good catch, I've readded it.

cdOut commented:
@parasharrajat PR should be ready for a recheck.

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal

parasharrajat reviewed (approved):
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

:ribbon: :eyes: :ribbon:  C+ reviewed

grgia commented:
Triggered a test build

grgia commented on src/CONST.ts:
```diff
@@ -2283,6 +2283,7 @@ const CONST = {
         PRIVATE_NOTES: 'privateNotes',
         DELETE: 'delete',
         MARK_AS_INCOMPLETE: 'markAsIncomplete',
+        BOOK_A_CALL: 'bookACall',
```
Where is this used?

cdOut replied:
This was a leftover from when Book a call was still in ReportDetailsPage, I've moved it to ProfilePage since so this can be removed, good catch.

grgia reviewed (commented):
Code looks good, just waiting on build to QA. Quick q!

cdOut committed:
remove unused const definition for BOOK_A_CALL

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal
merge main into @cdOut/three-dot-removal

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal

cdOut committed:
add isClosedReport check for holding expenses

cdOut committed:
fix hold promoted action not showing in specific cases

cdOut committed:
fix lint errors

cdOut commented:
@grgia We've found a bug with the Hold promoted action not showing for specific cases in the Details page that is currently on main, I've commited a fix here and included additional test steps for it in the PR description.

cdOut commented:
> We've found a bug with the Hold promoted action not showing for specific cases in the Details page that is currently on main, I've commited a fix here and included additional test steps for it in the PR description.

@parasharrajat Could you please retest this issue, especially the newly added test steps? We want to be sure this works and has been thoroughly tested.

trjExpensify commented:
@parasharrajat can we get that retested today?

trjExpensify commented:
I wrote them down here actually in the OP: https://github.com/Expensify/App/issues/44573#issue-2379105690

cdOut commented:
> @cdOut Where is the previous PR that you created for #44573

I have not created a PR for it yet.

trjExpensify commented:
Yep, works for me!


*Tom Rhys Jones *
*Expensify*


On Thu, 4 Jul 2024 at 19:35, Rajat ***@***.***> wrote:

> @cdOut <https://github.com/cdOut> Ok great. Then It will be better to add
> these changes to that PR and remain focused on the main issue. Currently,
> the Hold/Unhold button functionality has issues as I mentioned above which
> will be solved when that PR is created. I am happy to review that PR too.
>
> Anyways, I believe these changes are unrelated to the current issue we are
> solving in this PR.
>
> What do you say @trjExpensify <https://github.com/trjExpensify>?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Expensify/App/pull/44025#issuecomment-2209444124>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD3246IWJNM4RSGMSVLBD33ZKWIXBAVCNFSM6AAAAABJSQRYSSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMBZGQ2DIMJSGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

cdOut commented:
Agreed, I'll move those over there.

cdOut committed:
Revert "fix lint errors"

This reverts commit 02a3470e75a1ee5c84097ed3881e224f6fee9891.

revert hold changes

cdOut committed:
Revert "fix hold promoted action not showing in specific cases"

This reverts commit 2f10c65dca68e3f055a6126f9d029bb258ce647e.

revert hold changes

cdOut commented:
@parasharrajat I've reverted those two commits regarding the hold fix, I'll move it to the other issue.

cdOut commented on src/libs/ReportUtils.ts:
```diff
@@ -2805,7 +2805,7 @@ function canHoldUnholdReportAction(reportAction: OnyxInputOrEntry<ReportAction>)
     const isDeletedParentAction = isEmptyObject(parentReportAction) || ReportActionsUtils.isDeletedAction(parentReportAction);
 
-    const canHoldOrUnholdRequest = !isRequestSettled && !isApproved && !isDeletedParentAction;
+    const canHoldOrUnholdRequest = !isRequestSettled && !isApproved && !isDeletedParentAction && !isClosedReport(moneyRequestReport);
```
This was a change made because this check was added into MoneyRequestHeader and MoneyReportHeader logic for holds, and since the promoted action hold uses ReportUtils I also added it here for consistency. I would advise against reverting it here.

grgia replied:
@cdOut do you happen to know the background for why?

cdOut replied:
@grgia On main there was the possiblity to hold expenses which were already closed and it was not handled, hence why the `!isClosedReport(moneyRequestReport)` was added.

grgia replied:
> For consistency's sake and not removing this fix from the flow of the app I would not revert this one.

https://github.com/Expensify/App/pull/44656 cc @NikkiWines 


If we keep this check, shouldn't we update all of the `canHoldOrUnholdRequest` declarations (I see 3, ReportUtils, MoneyRequestHeader, MoneyReportHeader) to include this?

grgia replied:
@parasharrajat

grgia replied:
Oh NVM it's early, probably not the `MoneyRequestHeader`. So yeah this LGTM

cdOut replied:
I've reverted it just in case and I'll include this change in the PR fixing hold logic, should be good to merge.

cc: @grgia @parasharrajat

grgia commented:
@cdOut @parasharrajat let me know when this is ready for a final test build

cdOut committed:
Revert "add isClosedReport check for holding expenses"

This reverts commit c1abbb763d53f078fd02ae70868474d62337bbe5.

revert hold affecting commit

cdOut committed:
Merge branch 'main' into @cdOut/three-dot-removal
merge main into @cdOut/three-dot-removal

grgia reviewed (approved):
LGTM, tests well!

OSBotify commented:
:hand: This PR was not deployed to staging yet because QA is ongoing. It will be automatically deployed to staging after the next production release.

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/9837775807) to staging by https://github.com/grgia in version: 9.0.5-2 🚀

platform | result
---|---
🤖 android 🤖|success ✅
🖥 desktop 🖥|failure ❌
🍎 iOS 🍎|failure ❌
🕸 web 🕸|success ✅

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/9872741083) to production by https://github.com/Julesssss in version: 9.0.5-13 🚀

platform | result
---|---
🤖 android 🤖|success ✅
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅
🕸 web 🕸|success ✅

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/9945901401) to production by https://github.com/thienlnam in version: 9.0.6-8 🚀

platform | result
---|---
🤖 android 🤖|success ✅
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅
🕸 web 🕸|success ✅
